dev/io.openliberty.session.6.1.internal/resources/io/openliberty/session/resources/SessionMessages.nlsprops

```diff
@@ -0,0 +1,31 @@
+###############################################################################
+# Copyright (c) 2024 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License 2.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-2.0/
+# 
+# SPDX-License-Identifier: EPL-2.0
+###############################################################################
+#CMVCPATHNAME io.openliberty.session.6.1.internal/resources/io/openliberty/session/resources/SessionMessages.nlsprops
```

Comment:
Any reason didn't do the same as what was done for 6.0 to have the package name include `session61` instead of `session`?  If not exposing the nls classes, it would be good add `internal` to the package like was done with the 6.0 nls classes.